backend/homework/9/src/main/java/com/example/handson4/service/BookService.java

```diff
@@ -0,0 +1,65 @@
+package com.example.handson4.service;
+
+import com.example.handson4.constants.Constants;
+import com.example.handson4.dto.BookRequestDto;
+import com.example.handson4.dto.BookResponseDto;
+import com.example.handson4.entity.Book;
+import com.example.handson4.exception.custom.ObjectAlreadyExistsException;
+import com.example.handson4.exception.custom.ObjectNotFoundException;
+import com.example.handson4.mapper.BookMapper;
+import com.example.handson4.repository.BookRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Objects;
+import java.util.UUID;
+
+@Service
+public class BookService {
+    private final Logger logger = LoggerFactory.getLogger(BookService.class);
+    private BookRepository bookRepository;
+    @Autowired
+    public BookService(){
+        bookRepository = new BookRepository();
+    }
+
+    public BookResponseDto createBook(BookRequestDto bookRequestDto) {
```

Comment:
Make the code cleaner.
1. Proper indentation
2. Abstract out null checks and utils in different files